shared/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1726,6 +1725,8 @@ Undocumented.prototype.uploadExportFile = function( siteId, params ) {
 
 	req.upload.onprogress = params.onprogress;
 	req.onabort = params.onabort;
+
+	return req;
```

Comment:
Following the logic of `wpcom.js` and thinking through the future scenario where this endpoint is integrated into the library, I _think_ it should be okay to rely on the request object being returned [so long as it's not used as a Promise](https://github.com/Automattic/wpcom.js/blob/a6236f8c33f07c5b56f379e45370c9af747996e2/lib/util/send-request.js#L69-L76), which seems like a reasonable expectation.